docs/TBG_macros.cfg

```diff
@@ -266,25 +244,20 @@ TBG_openPMD="--openPMD.period 100   \
 # Create a checkpoint that is restartable every --checkpoint.period steps
 #   http://git.io/PToFYg
 TBG_checkpoint="--checkpoint.period 1000"
-# Select the backend for the checkpoint, available are openPMD and adios
-#    --checkpoint.backend openPMD
-#                         adios
-# Available backend options are exactly as in --openPMD.* and --adios.* and can be set
+# Select the backend for the checkpoint, available are openPMD
+#    --checkpoint.backend openPMa
```

Comment:
`openPMa` is a typo.

Also, why do we have the backend option at all now? The only value is openPMD. If it's left for compatibility or some reason like that, we should document it. But i would prefer to just remove the option alltogether, we can bring it back if needed in the future

Reply:
This option is optional and is by default `openPMD`. I like to keep this option to have a simpler life if we would introduce something else e.g. sionlib, a vendor library, .... Especially for checkpointing there is no strong need to use `openPMD`.

Reply:
That's correct. Yet the value in the example has a typo